.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
```

Comment:
Better to continue to run the CI on every pushed branch (provides an easy way to test things before creating a PR).

Reply:
okay, updated